packages/legend-extension-dsl-data-space/src/stores/query/DataSpaceTemplateQueryCreatorStore.ts

```diff
@@ -147,21 +147,9 @@ export class DataSpaceTemplateQueryCreatorStore extends QueryEditorStore {
       dataSpace,
       executionContext,
       (dataSpaceInfo: DataSpaceInfo) => {
-        if (dataSpaceInfo.defaultExecutionContext) {
-          this.applicationStore.navigationService.navigator.goToLocation(
-            generateDataSpaceTemplateQueryCreatorRoute(
-              guaranteeNonNullable(dataSpaceInfo.groupId),
-              guaranteeNonNullable(dataSpaceInfo.artifactId),
-              LATEST_VERSION_ALIAS, //always default to latest
-              dataSpaceInfo.path,
-              this.templateQueryId,
-            ),
-          );
-        } else {
-          this.applicationStore.notificationService.notifyWarning(
-            `Can't switch data space: default execution context not specified`,
-          );
-        }
+        this.applicationStore.notificationService.notifyWarning(
```

Comment:
why are we doing this change?

Reply:
This is to forbid switching dataspace when displaying/visiting a template query as you can't find this query under other dataspaces